core/commands/root.go

```diff
@@ -94,6 +94,7 @@ var rootSubcommands = map[string]*cmds.Command{
 	"mount":     MountCmd,
 	"name":      NameCmd,
 	"object":    ObjectCmd,
+	"patch":     PatchCmd,
```

Comment:
On Thu, May 28, 2015 at 10:39:32PM -0700, Jeromy Johnson wrote:
> > @@ -94,6 +94,7 @@ var rootSubcommands = map[string]*cmds.Command{
> >  	"mount":     MountCmd,
> >  	"name":      NameCmd,
> >  	"object":    ObjectCmd,
> > +	"patch":     PatchCmd,
> 
> @jbenet had mentioned that he thinks it would be good to have under
> the global namespace. Also, if we have to type object in there, the
> commands become absurdly long

I want more sugar on ‘ipfs patch’ (e.g. bubbling, see [1]).  I don't
expect anyone other than the very curious or scripters will be calling
the non-bubbly plumbing we're adding here.

[1]: https://github.com/ipfs/go-ipfs/pull/1187#issuecomment-106696791